packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -212,10 +212,7 @@
     <div class="signup--description">
       <Content slot-key="signup-description" />
       <div class="logo-wrapper">
-        <h4>Trusted by</h4>
-        <div class="partners-logo">
-          <img src="/img/authorization/proof-by.png" alt="" />
-        </div>
+        <CompanyLogos withHeading small v-bind:centered="false" />
```

Comment:
I think you only need the `v-bind:` for props if they are dynamic

Reply:
Without v-bind, it gets treated as a string. [Doc](https://vuejs.org/v2/guide/components-props.html#Passing-a-Boolean)

Reply:
Wow, love Vue